cmd/tunneld/tunneld.go

```diff
@@ -164,9 +169,36 @@ func startAutocert(opts *options, server *tunnel.Server, logger log.Logger) {
 		"addr", "80,443",
 	)
 	go func() {
-		fatal("failed to start HTTP: %s", http.ListenAndServe(":http", m.HTTPHandler(server)))
+		fatal("failed to start HTTP: %s", http.ListenAndServe(httpAddr, m.HTTPHandler(server)))
 	}()
 	go func() {
 		fatal("failed to start HTTPS: %s", s.ListenAndServeTLS("", ""))
 	}()
 }
+
+func hostPolicy(server *tunnel.Server) autocert.HostPolicy {
+	return func(_ context.Context, host string) error {
+		_, _, subscribed := server.Subscriber(host)
+		if !subscribed {
+			return fmt.Errorf("acme/autocert: host `%s` not subscribed", host)
+		}
+		return nil
+	}
+}
+
+func trimPort(hostPort string) (host string) {
```

Comment:
This looks like a duplicate.

Reply:
it will be removed as for no port validation when letsEncrypt is enabled